TLM/TLM/Manager/Impl/ExtVehicleManager.cs

```diff
@@ -1094,5 +1100,30 @@ public class ExtVehicleManager
             base.OnAfterLoadData();
             InitAllVehicles();
         }
+
+        /// <summary>
+        /// Checks if citizen should use electric car
+        /// </summary>
+        /// <param name="citizen">Citizen data</param>
+        /// <param name="citizenInstance">CitizenInstance data (spawned instance)</param>
+        /// <returns>true if should use electric car otherwise false</returns>
+        internal bool MustUseElectricCar(ref Citizen citizen, ref CitizenInstance citizenInstance) {
```

Comment:
maybe call this `ShouldUseElectricCar` car?

Reply:
... it was before